navigation/localization/auv_localization/package.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>auv_localization</name>
+  <version>0.0.0</version>
+  <description>perception and estimation of AUV using UAV</description>
+  <maintainer email="aryand@kth.se">aryan</maintainer>
+  <license>MIT</license>
+
```

Comment:
add the apt/pip-installable dependencies here.

Reply:
I'll commit this and you can give feedback on if it looks correct, unless there's someway i can test that myself

Reply:
the only good way to test is to freshly install this on a fresh OS too, we are doing a full-smarc2 install and compile automatically already. more of a housekeeping thing really.